pymatgen/io/lobster/outputs.py

```diff
@@ -433,27 +433,23 @@ def __init__(self, filename: str | None = None):
 
         # LOBSTER list files have an extra trailing blank line
         # and we don't need the header.
-        with zopen(filename, "rt") as f:
-            data = f.read().split("\n")[1:-1]
+        with zopen(filename, "rt") as file:
+            data = file.read().split("\n")[1:-1]
         if len(data) == 0:
             raise OSError("NcICOBILIST file contains no data.")
 
-        # If the calculation is spin polarized, the line in the middle
+        # If the calculation is spin-polarized, the line in the middle
         # of the file will be another header line.
-        if "spin" in data[len(data) // 2]:
-            # TODO: adapt this for orbitalwise case
-            self.is_spin_polarized = True
-        else:
-            self.is_spin_polarized = False
+        self.is_spin_polarized = "spin" in data[len(data) // 2]  # TODO: adapt this for orbitalwise case
 
         # check if orbitalwise NcICOBILIST
         # include case when there is only one NcICOBI
         for entry in data:  # NcICOBIs orbitalwise and non-orbitalwise can be mixed
             if len(data) > 2 and "s]" in str(entry.split()[3:]):
                 self.orbitalwise = True
                 warnings.warn(
-                    "This is an orbitalwise NcICOBILIST.lobster file. Currently, the orbitalwise information is not "
-                    "read!"
+                    "This is an orbitalwise NcICOBILIST.lobster file. "
```

Comment:
Is it possible to get orbital-wise interactions for multicenter ICOBIs?

Reply:
Yes, it is and the output format can be very complex with more than 3 centers or mix of n center interactions.

Reply:
Thanks 😃

Reply:
Also when Peter and I implemented the COBI, we never really thought of a high-throughput application, so the NcICOBIs are ordered as specified in the lobsterin... mixing different n center interactions up, therefore results in a very messed up NcICOBILIST.lobster file, especially when some are orbitalwise and other are not 🙈 need to check if and how that changed for lobster version 5 before implementing an orbitalwise feature here.

Reply:
😅